src/postamble.js

```diff
@@ -439,6 +434,9 @@ function exit(status, implicit) {
     }
 #endif // ASSERTIONS
   } else {
+#if !EXIT_RUNTIME && ASSERTIONS
+    checkUnflushedContent();
```

Comment:
Could this be done inside `exitRuntime` actually?

Reply:
Sure.. I agree that would be better.